draft-ietf-masque-connect-ip.md

```diff
@@ -1392,9 +1392,11 @@ based on these markings. However, the use of HTTP can limit the possibilities
 for differentiated treatment of the tunneled IP packets on the path between the
 IP proxying endpoints.
 
-If tunneled packets are subject to congestion control by the outer connection,
-the tunneled packets need to be treated equally regardless of their DSCP
-markings to not disrupt the congestion controller. In this scenario, the IP
+When a connection is congestion-controlled, marking packets with different DSCP
+can lead to reordering between them, and that can in turn lead the congestion
+controller to perform poorly. If tunneled packets are subject to congestion
```

Comment:
I would suggest to clarify that it is the "... in turn lead the HTTP connection's congestion controller to perform poorly."

Reply:
Done